### ISSUE TITLE
Facebook button handle

### ISSUE THREAD

**Issue by uc-asa** [magus/react-native-facebook-login#147]
Hello team,

My facebook button always come in center, I need to my facebook button in left side.

Plz suggest me how it possible

**Comment by lwhiteley** [comment]
sounds like its not the button itself but the styles of the container it is in.. ensure the styles are not `alignItems: center`

**Comment by lwhiteley** [comment]
only android has the feature to freely define the button to date 

https://github.com/magus/react-native-facebook-login/tree/master/android#step-6---include-in-javascript

**Comment by waleedarshad-vf** [comment]
Is there any way we can only change text without implementing whole fbLoginView?

**Comment by waleedarshad-vf** [comment]
yes need to pass 	facebookText={'FACEBOOK'}  with FBLogin but no mention in docs. atleast i did not find. btw Thanks

**Comment by lwhiteley** [comment]
ok good, will need to add to docs then
